src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
So I have looked into doing this and moving the build of PerfRunner over to be the same as RemoteExecutorConsoleApp.  I am hitting some snags because we need to reference xunit.performance.api.dll.  When I try and add that as a reference to the csproj it is unable to locate the dll, even with a hint path.  Looking at the build output it looks like the problem is that the SearchPaths is set only to look in the ref\netstandard directory.  Am I just doing something dumb here?